### DISCUSSION TITLE
How to remove the feature name from test results in Cypress Cloud?

### DISCUSSION THREAD

**Discussion by ghost** [badeball/cypress-cucumber-preprocessor#1202] [General]
Hi,
In Cypress Cloud test runs each result shows the feature name ("_As a user... I want to... So that.."_) followed by a `>` and then the scenario name. The scenario also gets truncated. This makes it harder to read and understand which scenario I'm looking at. Is there a way to configure the cucumber pre-propressor (or Cypress) so that only the scenario names appear, without the feature name/description? 
<img width="1322" alt="cypress cloud" src="https://github.com/badeball/cypress-cucumber-preprocessor/assets/173462513/16e5b8cc-43ef-4e08-bfbf-7e212ec6001b">

**Comment by badeball** [comment]
> Is there a way to configure the cucumber pre-propressor (or Cypress) so that only the scenario names appear, without the feature name/description?

There's no such configuration in the preprocessor, nor in Cypress, afaik.

Are you naming your features using user stories, IE. `Feature: As a ..` ?

**Reply by ghost** [reply]
> Are you naming your features using user stories, IE. Feature: As a .. ?

Yes.

e.g: `get_a_pizza.feature`
```gherkin
Feature: As a hungry QA, I want to get a pizza, So that I can make it through the day
  
Background:
    Given I am hungry

  Scenario: Domino's pizza
    Given I'm at home
    And I don't have a day off
    When I order a pizza online
    Then the pizza magically appears at my door
```

**Reply by badeball** [reply]
I'd put that in the description if I were you, like shown below.

```feature
Feature: Get a pizza

  As a hungry QA
  I want to get a pizza
  So that I can make it through the day
  
Background:
    Given I am hungry

  Scenario: Domino's pizza
    Given I'm at home
    And I don't have a day off
    When I order a pizza online
    Then the pizza magically appears at my door

**Reply by ghost** [reply]
I'll give it a try. Thank you very much! 🙇
